addons/account/tests/test_account_move_entry.py

```diff
@@ -74,6 +74,7 @@ def test_out_invoice_auto_post_at_date(self):
         # Create auto-posted (but not recurring) entry
         nb_invoices = self.env['account.move'].search_count(domain=[])
         self.test_move.auto_post = 'at_date'
+        self.test_move.checked = True
```

Comment:
this is weird - it should not have to be done manually for every auto-post entry
Are we missing `autocheck_on_post` on the journal?